src/main/java/org/jenkinsci/backend/ircbot/IrcBotImpl.java

```diff
@@ -276,6 +279,21 @@ private void handleDirectCommand(String channel, String sender, String login, St
         }
     }
 
+    private void sendFallbackMessage(String channel, String payload) {
+        if(StringUtils.containsIgnoreCase(payload, "thank")) {
+            sendMessage(channel, "You're welcome");
+        }
+        if(StringUtils.startsWithIgnoreCase(payload, "hello")) {
+            sendMessage(channel, "Hello you");
+        }
+        if(random.nextInt(3)==0) {
+            sendMessage(channel, "Wut?");
```

Comment:
Not sure what you mean :). You mean I should add many different answers and rotate on them randomly? Or that this answer should be rarer (here it'd be one time out of three).